src/io/aviso/taxi_toolkit/ui.clj

```diff
@@ -84,9 +84,8 @@
 
   [& el-val-or-entries]
   (let [el-val (if (= (count el-val-or-entries) 1)
-                 el-val-or-entries
+                 (first el-val-or-entries)
```

Comment:
No, if it's a just a map, the map needs to be treated as a collection. But here, when the only argument is one map, el-val-or-entries is a list with a map inside. This list however cannot be destructured with in the doseq.